.github/workflows/build.yml

```
@@ -67,3 +71,60 @@ jobs:
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          build-args: |
+            JAR_PATH=${{ env.JAR_PATH }}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)